Stack trace when printing an error message.

We were testing the provider, and tried to change a resource while logged in  with an API token that had read-only permissions. Instead of the error message at line 91 of `awx/resource_organization.go`, we got this stack trace:
```
Terraform will perform the following actions:

  # awx_organization.deleteme will be created
  + resource "awx_organization" "deleteme" {
      + default_environment = 0
      + description         = "just testing some more"
      + id                  = (known after apply)
      + max_hosts           = 0
      + name                = "Delete Me"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

awx_organization.deleteme: Creating...
╷
│ Error: Request cancelled
│ 
│   with awx_organization.deleteme,
│   on organizations.tf line 25, in resource "awx_organization" "deleteme":
│   25: resource "awx_organization" "deleteme" {
│ 
│ The plugin.(*GRPCProvider).ApplyResourceChange request was cancelled.
╵

Stack trace from the terraform-provider-awx_v0.24.3 plugin:

panic: interface conversion: interface {} is nil, not int

goroutine 55 [running]:
github.com/denouche/terraform-provider-awx/awx.resourceOrganizationsCreate({0xe293a8, 0xc0005274d0}, 0x0?, {0xbb8460?, 0xc000192000?})
	github.com/denouche/terraform-provider-awx/awx/resource_organization.go:91 +0x6d1
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc000173a40, {0xe293e0, 0xc0005269f0}, 0xd?, {0xbb8460, 0xc000192000})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/resource.go:778 +0x12e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc000173a40, {0xe293e0, 0xc0005269f0}, 0xc0004f8dd0, 0xc000528200, {0xbb8460, 0xc000192000})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/resource.go:909 +0xa7e
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc0003aded8, {0xe293e0?, 0xc0005268d0?}, 0xc00009bea0)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/grpc_provider.go:1072 +0xe8d
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc000259040, {0xe293e0?, 0xc0004efec0?}, 0xc00021f180)
	github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov5/tf5server/server.go:859 +0x574
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xcd29a0?, 0xc000259040}, {0xe293e0, 0xc0004efec0}, 0xc00021f110, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.19.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:467 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002521e0, {0xe2d0b8, 0xc0004069c0}, 0xc0004fc360, 0xc0003b7cb0, 0x131ad78, 0x0)
	google.golang.org/grpc@v1.57.1/server.go:1358 +0xe23
google.golang.org/grpc.(*Server).handleStream(0xc0002521e0, {0xe2d0b8, 0xc0004069c0}, 0xc0004fc360, 0x0)
	google.golang.org/grpc@v1.57.1/server.go:1735 +0xa36
google.golang.org/grpc.(*Server).serveStreams.func1.1()
	google.golang.org/grpc@v1.57.1/server.go:970 +0xca
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.57.1/server.go:981 +0x15c

Error: The terraform-provider-awx_v0.24.3 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

--- a/awx/resource_organization.go
+++ b/awx/resource_organization.go
@@ -88,7 +88,7 @@ func resourceOrganizationsCreate(ctx context.Context, d *schema.ResourceData, m 
 		diags = append(diags, diag.Diagnostic{
 			Severity: diag.Error,
 			Summary:  "Unable to create Organizations",
-			Detail:   fmt.Sprintf("Organizations with name %s in the project id %d, failed to create %s", d.Get("name").(string), d.Get("project_id").(int), err.Error()),
+			Detail:   fmt.Sprintf("Organizations with name %s, failed to create %s", d.Get("name").(string), err.Error()),
 		})
 		return diags
 	}

